src/components/DropdownItem/DropdownItem.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { Person } from '../../types/Person';
+
+type Props = {
+  person: Person | null;
```

Comment:
why person can be null. it prop comes from mapping people and looks like always will be Person